test/polyrith.lean

```diff
@@ -126,25 +126,25 @@ by test_polyrith "{\"data\":[\"(poly.const 1/1)\"],\"success\":true}" ["ff", "in
 
 example (x y : ℚ) (h1 : x*y + 2*x = 1) (h2 : x = y) :
   x*y = -2*y + 1 :=
-by test_polyrith "{\"data\":[\"(poly.const 1/1)\",\"(poly.const -2/1)\"],\"success\":true}" ["ff", "rat", "2", "[(((var0 * var1) + (2 * var0)) - 1), (var0 - var1)]", "((var0 * var1) - (((-1 * 2) * var1) + 1))"]  "linear_combination h1 - 2 * h2"
+by test_polyrith  "{\"data\":[\"(poly.const 1/1)\",\"(poly.const -2/1)\"],\"success\":true}" ["ff", "rat", "2", "[(((var0 * var1) + (2 * var0)) - 1), (var0 - var1)]", "((var0 * var1) - ((-2 * var1) + 1))"]  "linear_combination h1 - 2 * h2"
 
 example (x y : ℝ) (h1 : x + 2 = -3) (h2 : y = 10) :
   -y + 2*x + 4 = -16 :=
-by test_polyrith "{\"data\":[\"(poly.const 2/1)\",\"(poly.const -1/1)\"],\"success\":true}" ["ff", "real", "2", "[((var1 + 2) - (-1 * 3)), (var0 - 10)]", "((((-1 * var0) + (2 * var1)) + 4) - (-1 * 16))"]  "linear_combination 2 * h1 - h2"
+by test_polyrith  "{\"data\":[\"(poly.const 2/1)\",\"(poly.const -1/1)\"],\"success\":true}" ["ff", "real", "2", "[((var1 + 2) - -3), (var0 - 10)]", "(((-var0 + (2 * var1)) + 4) - -16)"]  "linear_combination 2 * h1 - h2"
```

Comment:
Is this supposd to output `+16`?

Reply:
It's irrelevant. This gets normalized without the user ever seeing it. It comes from the `lhs - rhs` pattern when `rhs` is -16.